udata/features/notifications/models.py

```diff
@@ -0,0 +1,42 @@
+from mongoengine import NULLIFY
+
+from udata.api_fields import field, generate_fields
+from udata.core.organization.notifications import MembershipRequestNotificationDetails
+from udata.core.user.api_fields import user_ref_fields
+from udata.core.user.models import User
+from udata.models import db
+from udata.mongo.datetime_fields import Datetimed
+from udata.mongo.queryset import UDataQuerySet
+
+
+class NotificationQuerySet(UDataQuerySet):
+    def with_organization_in_details(self, organization):
+        """This function must be updated to handle new details cases"""
+        return self(details__request_organization=organization)
+
+    def with_user_in_details(self, user):
+        """This function must be updated to handle new details cases"""
+        return self(details__request_user=user)
```

Comment:
In https://github.com/opendatateam/udata/pull/3524#discussion_r2537541294 query on generic embed fields seems to not work as intended… We may need to refactor this system when we add other details without `request_organization`… :-(